external_server/server_module/server_module.py

```
@@ -0,0 +1,83 @@
+        `connection_check` is a callback function used by the server module to check at least one device
+        is communicating with the external server.
+        """
+
+        self._id = module_id
+        self._api_adapter = _ApiAdapter(config=config, company=company, car=car)
+        try:
+            self._api_adapter.init()
+        except Exception as e:
+            msg = f"Module {module_id}: Error in init function. Check the configuration file. {e}"
+            logger.error(msg, car)
+            raise RuntimeError(msg)
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)